### DISCUSSION TITLE
archival feature

### DISCUSSION THREAD

**Discussion by JunkyardCat** [kaangiray26/geddit-app#48] [Ideas]
a way to import export post and comments. Also a way to not delete automatically post or update post since some comments get deleted later on. So im looking for more like archival purposes

**Comment by kaangiray26** [comment]
Which medium do you think is the best suitable option for saving the archive? Just saving them to the localStorage? to the phone filesystem? If you want to just view the post yourself I think that's fine.

But if you are thinking about sharing the archive with others there is a problem. Once the posts is downloaded on your phone it will be quite easy to edit the data and change the contents of the post/comment into whatever you like. After that there will be no way to prove the integrity of the shared archive data. There should also be a way to verify it via cryptographic hashes etc.

**Reply by JunkyardCat** [reply]
Theres really no need to worry about the integrity since it should be the responsibility of the user themselves to check if the archived data are trustworthy or not
